[Getting Started Docs][A/B test] Create Snippets from install docs (#6397)
PostHog/posthog.com: contents/docs/integrate/_snippets/install-react.mdx

> For installation with Next.js we'd recommend following the [Next.js integration guide](/docs/integrate/next-js) instead.

Reviewer: ```suggestion
> For Next.js, we recommend following the [Next.js integration guide](/docs/integrate/next-js) instead.
```
